README.md

```diff
@@ -1 +1,26 @@
-# hw01
\ No newline at end of file
+# Homework #1
+##### by
+### David Johnson
+
+Hello world!
+
+I am David, a MAPSS student looking to concentrate in Comparative Human Development, and even **more** specifically than that, adolescent development.
+I am originally from [Florida](https://time.com/4214021/florida-man-throws-alligator-into-wendys-drive-thru-window/), went to school in Tennessee - just not *at* Tennessee ([hint](https://www.espn.com/college-football/recap?gameId=333272633)), and have been living in [California](https://www.youtube.com/watch?v=uqSKl7sdUa8) for the past 4 years.
+
+#### Things I Like
+* Improv comedy
+* Cats
+* Learning to code
+* Donating hair! Here is an 'after' photo I took last January:
+
+![](After_Haircut_Rotated.jpg)
+
+
+To see the 'before' photo, click [here](Before_Haircut_Rotated.jpg)!
```

Comment:
The linked photo opens very zoomed in on one corner. I was able to find a way to make it fit entirely inside the screen when opened, putting

{r  out.width = "20%"}
include_graphics(img1_path)

before the linked line. This did also shrink the entire Markdown display, though, and I was unable to find a way to shrink only the linked photo.

Reply:
Yes! I discovered this during Dr. Soltoff's office hours, and the general conclusion seemed to be, "Yeah, there _are_ ways to address this...but that would probably be a bit beyond the scope of what we're doing at the moment." So we left it as-is. Unfortunately it makes for a much less dramatic grand reveal! Thank you for this tip, though.